[2023-12-19 22:00:04,762] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-19T21:00:00+00:00 [queued]>
[2023-12-19 22:00:04,894] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-19T21:00:00+00:00 [queued]>
[2023-12-19 22:00:04,901] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-12-19 22:00:04,915] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2023-12-19 22:00:04,990] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-12-19 22:00:05,057] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): fill> on 2023-12-19T21:00:00+00:00
[2023-12-19 22:00:05,075] {standard_task_runner.py:52} INFO - Started process 74947 to run task
[2023-12-19 22:00:05,085] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_python_weather_v01', 'fill', '2023-12-19T21:00:00+00:00', '--job-id', '184', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_python_weather.py', '--cfg-path', '/tmp/tmppqx__2fb', '--error-file', '/tmp/tmpzm038gb_']
[2023-12-19 22:00:05,092] {standard_task_runner.py:77} INFO - Job 184: Subtask fill
[2023-12-19 22:00:05,204] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_with_python_weather_v01.fill 2023-12-19T21:00:00+00:00 [running]> on host 5f3722f1fc3b
[2023-12-19 22:00:05,388] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ggoni
AIRFLOW_CTX_DAG_ID=dag_with_python_weather_v01
AIRFLOW_CTX_TASK_ID=fill
AIRFLOW_CTX_EXECUTION_DATE=2023-12-19T21:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-19T21:00:00+00:00
[2023-12-19 22:00:07,279] {python.py:118} INFO - Done. Returned value was: None
[2023-12-19 22:00:07,315] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=dag_with_python_weather_v01, task_id=fill, execution_date=20231219T210000, start_date=20231219T220004, end_date=20231219T220007
[2023-12-19 22:00:07,360] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-19 22:00:07,496] {local_task_job.py:146} INFO - Task exited with return code 0
